index.html

```diff
@@ -173,5 +173,18 @@ <h5>Skills</h5>
             </li>
         </ul>
     </section>
+    <div class="form-container">
```

Comment:
- [ ] please add a background image to the contact section so it looks like the figma version.